rllib/algorithms/algorithm.py

```diff
@@ -1326,7 +1339,14 @@ def training_step(self) -> ResultDict:
             "timestep": self._counters[NUM_ENV_STEPS_SAMPLED],
         }
         with self._timers[SYNCH_WORKER_WEIGHTS_TIMER]:
+            # TODO (Avnish): Implement this on trainer_runner.get_weights().
+            # TODO (Kourosh): figure out how we are going to sync MARLModule
+            # weights to MARLModule weights under the policy_map objects?
+            src_weights = None
```

Comment:
appreciate the change here. looks much cleaner.
can we rename this to from_worker, instead of src_weights please?

Reply:
done.